stable/aws-sigv4-proxy-admission-controller/values.yaml

```diff
@@ -0,0 +1,18 @@
+name: sidecar-injector
+
+replicaCount: 1
+
+image:
+  uri: public.ecr.aws/aws-observability/aws-sigv4-proxy-admission-controller:1.0
+  pullPolicy: IfNotPresent
+
+env:
+  awsSigV4ProxyImage: public.ecr.aws/aws-observability/aws-sigv4-proxy:1.0
+
+serviceAccount:
+  create: true
+  name: ""
+
+service:
```

Comment:
You need to document the values in this file.
https://helm.sh/docs/chart_best_practices/values/#document-valuesyaml

Reply:
Done.